src/components/LogRow.vue

```diff
@@ -62,6 +62,7 @@
   shrink-0
   text-right
   pl-1
+  overflow-auto
```

Comment:
I believe tailwind has a `truncate` utility for adding overflow ellipsis styles. If instead you want the user to be able to scroll this small area, then g2g, might get weird on devices with bulky scroll bars though.